orderer/common/cluster/util.go

```diff
@@ -567,31 +567,46 @@ func globalEndpointsFromConfig(aggregatedTLSCerts [][]byte, bundle *channelconfi
 	return globalEndpoints
 }
 
-type BlockVerifierFunc func(header *common.BlockHeader, metadata *common.BlockMetadata) error
+// BFTEnabledInConfig takes a config block as input and returns true if consenter type is BFT
+func BFTEnabledInConfig(block *common.Block, bccsp bccsp.BCCSP) bool {
+	bftEnabled, _, _, _ := getConsentersAndPolicyFromConfigBlock(block, bccsp)
```

Comment:
you silently ignore the error

Reply:
ignored it , to defer its handling, so that it be can handled by block verifier

Reply:
but you don't handle it, you just log the error. 

I think that if you cannot extract such fundamental information from the config block, then just do not instantiate the block fetcher.

Reply:
sure. I was not sure whether to handle this error here.